src/backend/devices/kpro/constants.py

```diff
@@ -6,6 +6,11 @@
 KPRO23_ID = 23  # v2 and v3 works equally so I mixed both version numbers
 KPRO4_ID = 4
 
+# s300 device ids
+S300V3_ID_VENDOR = 0x1C40
```

Comment:
just to follow the same format as in kpro I would consider apply the same one for s300 `S3003_ID_VENDOR`